domains/kuyo.json

```
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "KaisenyiKuyo",
+           "email": "",
+           "discord": "1033580960882626630",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.bbKXmCiGxopzxRAn8y4AwSNKZPn8mcP5oBcBGiw7MqOboj2An5MwKK_yvFPC7qczxkFP2qdcqm-nz5mCumyJc11udZcmps4xZ9sfV6NtP6U45L6zwtRPmbvoMT3zcpceff510yy8F1V_nBK5AO-oqs_geVobQNr9WblM4i5TCi9VaLt15Rz-27yYx8zjPRVjM1fnvlHNokEBZCaRmfvufFIAj84fnZIKsTf73VhQYgdOqhkkesogyOQycINCv3nc6QgXIPIRHl3HQ--iSn9oGNc4kj_ZmDnUz5Ss901EJ86RQHsIS2P7C_I45Ml3nsRKUP0fZPViuamVCk8uI3C1qg.-8JefwiXKms1eWqNsqECww.6KSX8BVO-mTOIsbMyC1nMtHgq5AtWJzedu9unpdp4yRLtpSSFMcdGAzUxXvHOroIXgGbe3UcUmRndFTtVequhtNYJ3oTXKq3QWiaL7prlBo.-vGFp-rFsrtDBQCaEP1eUw"
+        },
+    
+        "record": {
+            "TXT": "kaisenyikuyo.github.io"
```

```suggestion
            "CNAME": "kaisenyikuyo.github.io"
```
This is a TXT record, it's supposed to be a CNAME record.